dts/bindings/wifi/inventek,eswifi.yaml

```diff
@@ -0,0 +1,45 @@
+#
+# Copyright (c) 2018, I-SENSE group of ICCS
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+---
+title: Inventek eS-WiFi WiFi module
+id: inventek,eswifi
+version: 0.1
+
+description: >
+    This binding gives the base representation of esWiFi WiFi module
+
+inherits:
+    !include spi-device.yaml
+
+properties:
+    compatible:
+      constraint: "inventek,eswifi"
+
+    resetn-gpios:
```

Comment:
Could you add a small description to all *-gpios properties?

Reply:
sure

Reply:
done